packages/jane/rollup.config.js

```diff
@@ -0,0 +1,43 @@
+import url from '@rollup/plugin-url'
```

Comment:
remove this file

Reply:
Actually this entire `packages` folder should not be in the PR

Reply:
Im not sure how to do everything youre requesting but working through what I do know how to do :) Whole folder is removed